packages/clay-css/src/scss/functions/_lx-icons-generated.scss

```diff
@@ -34,6 +34,14 @@
 
 	'analytics': '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path class="analytics-bar1 lexicon-icon-outline" d="M96 415.8H32c-17.7 0-32 14.3-32 32v32c0 17.7 14.3 32 32 32h64c17.7 0 32-14.3 32-32v-32c0-17.7-14.3-32-32-32z" fill="#{$color}"/><path class="analytics-bar2 lexicon-icon-outline" d="M288 319.8h-64c-17.7 0-32 14.3-32 32v128c0 17.7 14.3 32 32 32h64c17.7 0 32-14.3 32-32v-128c0-17.7-14.3-32-32-32z" fill="#{$color}"/><path class="analytics-bar3 lexicon-icon-outline" d="M480 223.8h-64c-17.7 0-32 14.3-32 32v224c0 17.7 14.3 32 32 32h64c17.7 0 32-14.3 32-32v-224c0-17.7-14.3-32-32-32z" fill="#{$color}"/><path class="analytics-uptrend-arrow lexicon-icon-outline" d="M415 110.2v23.6c3.1 44.2 64.6 41.3 64.8 0V32.6c0-17.9-13.2-32.4-32.4-32.4H346.1c-42.8 2.3-43.2 61.9 0 64.8h23.6l-95.3 94.9-114.2-.2h-.1c-10.1 0-19.5 4.7-25.6 12.8L38.6 300.3c-24.2 35.4 25.4 70.9 51.2 38.4l86.5-115.1 111.4.2h.1c8.5 0 16.6-3.4 22.6-9.3L415 110.2z" fill="#{$color}"/></svg>',
 
+	'angle-double-left-small': '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path class="lexicon-icon-line-1" d="m225.627 407.104-131.524-129.6c-20.142-18.642-17.322-28.161-.303-43.829l131.827-129.898c24.674-23.203 62.54 10.71 37.967 33.814L143.752 255.986l119.137 117.107c26.386 24.889-10.272 59.396-37.262 34.011Z"/><path class="lexicon-icon-line-2" d="m385.627 407.104-131.524-129.6c-20.142-18.642-17.322-28.161-.303-43.829l131.827-129.898c24.674-23.203 62.54 10.71 37.967 33.814L303.752 255.986l119.137 117.107c26.386 24.889-10.272 59.396-37.262 34.011Z"/></svg>',
```

Comment:
@pat270 I don't remember very well but do we need to manually add the `fill="#{$color}"` property?

Reply:
@matuzalemsteles it should generate it when we do `yarn compile` in `packages/clay-css`. I'll double check and update this pr.